### PR TITLE
Reorganize ReplacingDisks.md to avoid confusion

### PR DESCRIPTION
**Pull request by Aviatorpaal**
Needs more information from the ixSystems team describing replacing disks in non-hot swap systems

Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

**Comment by DjP-iX** [comment]
@Aviatorpaal sorry for your confusion with this tutorial, but thanks for bringing it to our attention. I've reorganized the existing content a bit to hopefully make it a bit easier to follow the process for systems with or without a hot-spare.

File: content/SCALETutorials/Storage/Disks/ReplacingDisks.md [hunk 1]
```diff
@@ -26,6 +26,21 @@ TrueNAS wipes the data on the replacement disk as part of the process.
 Disk replacement automatically triggers a pool resilver.
 {{< /hint >}}
 
+This tutorial includes instructions for replacing a failed disk in TrueNAS systems with and without an available hot spare.
```
Comment (Aviatorpaal) [inline content/SCALETutorials/Storage/Disks/ReplacingDisks.md:29]: What if the users have systems which do not allow hot plug or hot swapping their JBOD HDDs? There should be a section describing the "offlining", detaching and rebooting process in this case, as not all will do this with their systems operational.

File: content/SCALETutorials/Storage/Disks/ReplacingDisks.md [hunk 2]
```diff
@@ -134,21 +133,21 @@ Click **Detach** on the **ZFS Info** widget on the **Devices** screen for the di
...
+If you have a hot spare inserted into the pool and then follow the instructions in [Replacing a Failed Disk Without a Hot Spare](#replacing-a-failed-disk-without-a-hot-spare), TrueNAS automatically returns the hot spare disk to the existing **Spare** vdev and **ONLINE** status.
 
 However, we do not recommend this method, because it causes two resilver events: one when activating the hot spare and again when replacing the failed disk.
 Resilvering degrades system performance until completed and causes unnecessary strain on the disk.
 
-To avoid unnecessary resilvers, [promote the hot spare](#detaching-a-failed-disk) then recreate the hot spare vdev.
+To avoid unnecessary resilvers, promote the hot spare by [detaching the failed disk]](#detaching-a-failed-disk) then recreate the hot spare vdev.
 {{< /expand >}}
 
-WHAT IF I DO NOT HAVE A HOT SPARE/A SYSTEM WHICH DOES NOT SUPPORT HOT SWAPPING? <<< Offline/reboot process described here >>>
```
Comment (Aviatorpaal) [inline content/SCALETutorials/Storage/Disks/ReplacingDisks.md]: In which sequence should the steps be completed for the user if their system does not allow hot plug or hot swapping drives?

File: content/SCALETutorials/Storage/Disks/ReplacingDisks.md
Comment (Aviatorpaal) [inline content/SCALETutorials/Storage/Disks/ReplacingDisks.md:1]: Looks great! Thanks for cleaning it up

**Review by Aviatorpaal — COMMENTED** [review]
Looking good! I still think there is one thing missing: a section for users having non-hot plug or non-hot swappable drives

**Comment by Mrt134** [comment]
Runner is stuck, so reviewed locally - no issues and approving for merge.

**Pull request merged**